Vocab namespaces section

stain opened the pull request:
This fixes #193 by adding explicit text to the 1.1 Namespaces section.

It also includes direct links to the OWL version of the ontology (depending on issue #66).

azaroth42 commented:
There's lots of changes in index-respec.html that don't seem to be reflected in index-linktemplate ?

stain commented:
I think that is because @iherman had edited index-linktempalte without regenarating index-respec.html

stain committed:
Regenerated from index-linktemplate.html

using
    python make_links.py

and then saving index-respec.html from Firefox to index.html

stain committed:
Added Namespace section

Also corrected editor draft and previous URLs

stain commented:
OK - now I did it in new commits - first a simple commit that just regenerates index-respec.html and index.html (lovely thing to have build artifacts in git, huh..) - and then added my 1.1 changes as a single commit.

stain commented:
BTW I added the `index*` regeneration (now that I think I get it) to the `Makefile` of the #188 pull request - it will ask you to save from the browser when index.html is out of dat.e

stain commented:
In commit b2c83235b719d9896877005ada7aa304d8174ba1 @iherman said:

>    (I edited the nametemplate files and the index.html manually, did not bother with regeneration; wasn't even sure how to do that for the vocab file, there is no relevant README)

(I added such `README.md` to #188 )

So now his changes are reflected in my commit regenerating in d67c47d031d87aba4a6855f6023fa03471886c47

(I edited this comment as I got it the wrong way around first)

azaroth42 commented:
These PRs are now very stale :( I think it's easier to generate an ontology from the vocab at this stage, to ensure that they're synchronized.  So I'm closing these PRs, sorry!
